proc/proc_test.go

```diff
@@ -1475,3 +1477,64 @@ func TestIssue384(t *testing.T) {
 		assertNoError(err, t, "EvalVariable()")
 	})
 }
+
+func TestIssue332_Part1(t *testing.T) {
+	// Next shouldn't step inside a function call
+	withTestProcess("issue332", t, func(p *Process, fixture protest.Fixture) {
+		start, _, err := p.goSymTable.LineToPC(fixture.Source, 8)
+		assertNoError(err, t, "LineToPC()")
+		_, err = p.SetBreakpoint(start)
+		assertNoError(err, t, "SetBreakpoint()")
+		assertNoError(p.Continue(), t, "Continue()")
+		assertNoError(p.Next(), t, "first Next()")
+		locations, err := p.CurrentThread.Stacktrace(2)
+		assertNoError(err, t, "Stacktrace()")
+		if locations[0].Call.Fn == nil {
+			t.Fatalf("Not on a function")
+		}
+		if locations[0].Call.Fn.Name != "main.main" {
+			t.Fatalf("Not on main.main after Next: %s (%s:%d)", locations[0].Call.Fn.Name, locations[0].Call.File, locations[0].Call.Line)
+		}
+		if locations[0].Call.Line != 9 {
+			t.Fatalf("Not on line 9 after Next: %s (%s:%d)", locations[0].Call.Fn.Name, locations[0].Call.File, locations[0].Call.Line)
+		}
+	})
+}
+
+func TestIssue332_Part2(t *testing.T) {
+	// Step should skip a function's prologue
+	// In some parts of the prologue, for some functions, the FDE data is incorrect
+	// which leads to 'next' and 'stack' failing with error "could not find FDE for PC: <garbage>"
+	// because the incorrect FDE data leads to reading the wrong stack address as the return address
+	if disableFailingTests {
```

Comment:
Add a TODO here to enable or remove this test once 1.6 lands, since that seems to fix the FDE issue.